pages/demo/html/elementul-hr.tsx

```diff
@@ -5,33 +5,23 @@ import Demo, { DemoPreview } from '~/components/demo';
 import Header from '~/components/Header';
```

Comment:
Same as in the previous files.

We should only modify where to replace with `<FormattedText>`.